src/lib/AuthButton.svelte

```diff
@@ -0,0 +1,53 @@
+<script lang="ts">
+  import { onMount } from "svelte";
+  import { auth, pb } from "$lib/stores/authStore";
+  import { goto } from "$app/navigation";
+
+  let isAuthenticated = false;
+
+  onMount(auth.subscribe((value) => {
+    isAuthenticated = value.isAuthenticated;
+  }));
+
+  function goHome() {
```

Comment:
nitpick: Dette kan for øvrig ekstraheres til en utility function

Reply:
Jeg tenkte på å gjøre dette, men da den refererer til variablen `isAuthenticated`, som er utenfor scopet av selve funksjonen, fant jeg ikke noen god måte å gjøre dette på.